bin/direxec.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+
+mode = os.path.basename(__file__)
+
+dir_odoo = '/opt/odoo'
+dir_common = os.path.join(dir_odoo, 'common', '%s.d' % mode)
+dir_custom = os.path.join(dir_odoo, 'custom', '%s.d' % mode)
+
+files = [
+    os.path.join(dir_common, d) for d in os.listdir(dir_common)
+]
+
+try:
+    files += [
+        os.path.join(dir_custom, d) for d in os.listdir(dir_custom)
+    ]
+except OSError:
+    pass
+
+for command in sorted(files):
+    if os.access(command, os.X_OK):
+        subprocess.check_call(command)
+
+extra_command = sys.argv[1:]
+
+if extra_command:
+    os.execvp('odoo', extra_command)
```

Comment:
Change `'odoo'` and put `extra_command[0]`.

We can also add a little smartypants feature here and do:

```python
if extra_command[0] == 'shell' or extra_command[0].startswith("-"):
    extra_command.insert(0, "odoo")
```

That would let us run commands like `docker-compose run --rm odoo shell` or `docker-compose run --rm odoo --version` instead of `docker-compose run --rm odoo odoo shell` and `docker-compose run --rm odoo odoo --version` (saving 1 `odoo`). Not a requirement though.